CreateSnapshot/src/main/java/com/rfs/CreateSnapshot.java

```diff
@@ -65,6 +71,13 @@ public static class Args {
                 required = false,
                 description = "The maximum snapshot rate in megabytes per second per node")
         public Integer maxSnapshotRateMBPerNode;
+
+        @Parameter(required = false,
```

Comment:
Also lets use a standard format for arguments, in this case `--otel-collector-endpoint`

Reply:
Ugh - I've changed this for now, but we have to switch everything over to be camel case.  This will be a bug waiting to happen.  See https://opensearch.atlassian.net/browse/MIGRATIONS-1837